.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: true
```

Comment:
That's another reason to use https://github.com/PHP-FFMpeg/PHP-FFMpeg, in that case this can easily be mocked, and we don't have to do this. Then the tests can still be ran with the new travis infrastructure.